transform/panic.go

```diff
@@ -0,0 +1,32 @@
+package transform
+
+import (
+	"tinygo.org/x/go-llvm"
+)
+
+// ReplacePanicsWithTrap replaces each call to panic (or similar functions) with
+// calls to llvm.trap, to reduce code size. This is the -panic=trap command-line
+// option.
+func ReplacePanicsWithTrap(mod llvm.Module) {
+	builder := llvm.NewBuilder()
```

Comment:
This should use a context. Maybe it should be `mod.Context().NewBuilder()`?

Reply:
Good catch! Fixed now.